cmd/aries-agent-mobile/pkg/wrappers/command/aries.go

```diff
@@ -100,11 +99,7 @@ func prepareFrameworkOptions(opts *config.Options) ([]aries.Option, error) {
 		options = append(options, aries.WithTransportReturnRoute(opts.TransportReturnRoute))
 	}
 
-	if opts.DBNamespace != "" {
-		options = append(options, defaults.WithStorePath(opts.DBNamespace))
-	} else {
-		options = append(options, aries.WithStoreProvider(mem.NewProvider()))
-	}
+	options = append(options, aries.WithStoreProvider(mem.NewProvider()))
```

Comment:
Note: the mobile wrapper won't really be usable until we can have a real storage implementation injected.